Upstream: Not yet, but can be.
Description: Don't throw away errors from os-prober; it makes things hard to
 debug.

diff -Nur -x '*.orig' -x '*~' grub2-1.97~beta3/util/grub.d/30_os-prober.in grub2-1.97~beta3.new/util/grub.d/30_os-prober.in
--- grub2-1.97~beta3/util/grub.d/30_os-prober.in	2009-09-18 18:44:44.000000000 +0100
+++ grub2-1.97~beta3.new/util/grub.d/30_os-prober.in	2009-09-18 18:46:19.000000000 +0100
@@ -73,7 +73,7 @@
   exit 0
 fi
 
-OSPROBED="`os-prober 2> /dev/null | tr ' ' '^' | paste -s -d ' '`"
+OSPROBED="`os-prober | tr ' ' '^' | paste -s -d ' '`"
 if [ -z "${OSPROBED}" ] ; then
   # empty os-prober output, nothing doing
   adjust_timeout
